### PR TITLE
Add retired status

### PR DESCRIPTION
**Pull request by crm-skyach**
Display address from active wallets

Generate address from active wallets

deposit/withdraw from retired wallets

Add status for address

### CODE REVIEW

**Comment by chumaknadya** [comment]
Could you add tests for this PR?

File: app/models/deposit.rb [hunk 1]
```diff
@@ -183,6 +183,7 @@ def as_json_for_event_api
       currency:                 currency_id,
       amount:                   amount.to_s('F'),
       state:                    aasm_state,
+      status:                   PaymentAddress.find_by_address(address)&.wallet&.status,
```
Comment (calj) [inline app/models/deposit.rb]: This looks wrong to me. Why would we need the status of the deposit address to be exposed for a deposit?

Comment (chumaknadya) [inline app/models/deposit.rb]: its very confusing to have state and status here, and its not deposit status, 
its more like wallet_state or payment_address_state

**Review by chumaknadya — CHANGES_REQUESTED** [review]

File: app/models/deposit.rb (hunk 1 above)
Comment (crm-skyach) [reply app/models/deposit.rb]: Hi @chumaknadya i have changed the status to wallet_state. Please check the comment @calj do we need to remove or do you have any other suggestions on that.

**Comment by crm-skyach** [comment]
> Could you add tests for this PR?

I have added the test cases

**Pull request closed without merging**